src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerSqlDependencyExtensions.cs

```diff
@@ -358,5 +359,74 @@ public static partial class ILoggerExtensions
                 isSuccessful: isSuccessful,
                 context: context));
         }
+
+        /// <summary>
+        /// Logs a SQL dependency.
+        /// </summary>
+        /// <param name="logger">The logger to track the SQL dependency.</param>
+        /// <param name="connectionString">The SQL connection string.</param>
+        /// <param name="sqlCommand">The pseudo SQL command information that gets executed against the SQL dependency.</param>
+        /// <param name="operationName">The functional description of the operation that was performed on the SQL database.</param>	
+        /// <param name="isSuccessful">The indication whether or not the operation was successful.</param>
+        /// <param name="measurement">The measuring the latency to call the SQL dependency.</param>
+        /// <param name="dependencyId">The ID of the dependency to link as parent ID.</param>
+        /// <param name="context">The context that provides more insights on the dependency that was measured.</param>
+        /// <exception cref="ArgumentNullException">Thrown when the <paramref name="logger"/> is <c>null</c>.</exception>
+        /// <exception cref="ArgumentException">Thrown when the <paramref name="connectionString"/> is blank.</exception>
+        public static void LogSqlDependencyWithConnectionString(
```

Comment:
Just wondering why the name has changed to ...WithConnectionString ?   Is it to avoid name clashes ?

Reply:
Yeah. There was already a small clash where you would need to explicitly use the `connectionString` parameter name to void it being the `serverName` instead of other overloads; and if we would add these, we would have even more clashes. So, decided it would be better anyway to have another method name for this.